public/index.html

```diff
@@ -5,9 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
+    <!-- Bootstrap CSS CDN -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.typekit.net/vlg8jnr.css">
     <title>vue-app</title>
   </head>
   <body>
+    <!-- JQuery for Bootstrap -->
```

Comment:
Was there ever a concern about using JQuery? Vaguely remember something coming up

Reply:
There are functionalities of bootstrap that only work with JQuery. I added originally them in case we need them in the future but just removed them cause nothing on the requirements menu needs it. Yea... I'm pretty sure it's generally bad to use JQuery with another framework